ci/terraform/shared/kms.tf

```diff
@@ -333,3 +333,34 @@ resource "aws_kms_alias" "orchestration_to_auth_signing_key_alias" {
   name          = "alias/${var.environment}-orch-signing-key-alias"
   target_key_id = aws_kms_key.orchestration_to_auth_signing_key.key_id
 }
+
+# Authorization Code store Signing KMS key
+
+resource "aws_kms_key" "auth_code_store_signing_key" {
+  description              = "KMS signing key for Authorization code store in DynamoDB"
+  deletion_window_in_days  = 30
+  key_usage                = "SIGN_VERIFY"
+  customer_master_key_spec = "ECC_NIST_P256"
+  policy                   = data.aws_iam_policy_document.auth_code_kms_policy_document.json
+  tags                     = local.default_tags
+}
+
+resource "aws_kms_alias" "auth_code_signing_key_alias" {
+  name          = "alias/${var.environment}-auth-code-signing-key-alias"
+  target_key_id = aws_kms_key.auth_code_store_signing_key.key_id
+}
+
+data "aws_iam_policy_document" "auth_code_kms_policy_document" {
+  count = var.use_localstack ? 0 : 1
```

Comment:
No need for count = anymore, localstack isn't ever going to be used and it complicates things.